client/service/src/client/wasm_override.rs

```diff
@@ -226,11 +232,11 @@ mod tests {
 	#[test]
 	fn should_get_runtime_version() {
 		let wasm = WasmBlob::new(substrate_test_runtime::wasm_binary_unwrap().to_vec());
-		let executor =
-			NativeExecutor::<LocalExecutor>::new(WasmExecutionMethod::Interpreted, Some(128), 1);
+		let executor = NativeExecutor::<LocalExecutor>::new(WasmExecutionMethod::Interpreted, 1);
 
-		let version = WasmOverride::runtime_version(&executor, &wasm, Some(128))
-			.expect("should get the `RuntimeVersion` of the test-runtime wasm blob");
+		let version =
+			WasmOverride::runtime_version(&executor, &wasm, Some(128), CodeContext::Consensus)
```

Comment:
Does wasm override apply to offchain?
If so will it run with consensus context?